src/main/java/com/thebuzzmedia/exiftool/ExifTool.java

```diff
@@ -525,6 +526,41 @@ private Map<Tag, String> getImageMeta(File image, Collection<? extends Tag> tags
 		return tagHandler.getTags();
 	}
 
+	/**
+	 * Run users custom Exiftool command on image and returns raw output from Exiftool as string
+	 * This just passes the arguments to Exiftool and does not do any checking on the validity of
+	 * those arguments before passing them to Exiftool. The user is also responsible for parsing
+	 * the raw output that has been returned.
+	 *
+	 * @param image Image.
+	 * @return String with whatever exiftool outputs.
+	 * @throws IOException If something bad happen during I/O operations.
+	 * @throws NullPointerException If one parameter is null.
+	 * @throws com.thebuzzmedia.exiftool.exceptions.UnreadableFileException If image cannot be read.
+	 */
+	public String getRawExifToolOutput(File image, List<String> arguments) throws IOException {
+		requireNonNull(image, "Image cannot be null and must be a valid stream of image data.");
+		requireNonNull(arguments, "Arguments cannot be null.");
+		isReadable(image, String.format("Unable to read the given image [%s], ensure that the image exists at the given withPath and that the executing Java process has permissions to read it.", image));
+
+		int expectedSize = arguments.size() + 2;
+		List<String> args = new ArrayList<>(expectedSize);
+
+		addAll(args, arguments);
+
+		// Add image argument.
+		args.add(image.getAbsolutePath());
+
+		// Add last argument.
+		// This argument will only be used by exiftool if stay_open flag has been set.
+		args.add("-execute");
```

Comment:
What happends if `-execute` is already in the `arguments` list?

Reply:
This is a very good question and it got me thinking it this wasn't limiting the scope of the commands a user could enter.  And you are right why be adding "-execute" when the whole idea is to expose the ability for a user to enter their own exiftool command.   ok so I updated the PR and updated the tests.

I took out passing in the image file.  So this way if I want a command where I use exiftool to copy certain metadata from one image to to another I can use this method.  

What do you think about this?

Reply:
@mjeanroy just wanted to check back in on this.

Reply:
Make sense, so this function is more a helper function on top of this exiftool wrapper :)